db/structure.sql

```diff
@@ -357,48 +357,6 @@ CREATE INDEX index_taxonomy_licenses_on_entity_type_and_entity_id ON taxonomy_li
 
 
 --
--- Name: organizations_to_tsvector_idx; Type: INDEX; Schema: public; Owner: -; Tablespace: 
---
-
-CREATE INDEX organizations_to_tsvector_idx ON organizations USING gin (to_tsvector('english'::regconfig, (organization_name_legal_business_name)::text));
-
-
---
--- Name: organizations_to_tsvector_idx1; Type: INDEX; Schema: public; Owner: -; Tablespace: 
---
-
-CREATE INDEX organizations_to_tsvector_idx1 ON organizations USING gin (to_tsvector('english'::regconfig, (other_organization_name)::text));
-
-
---
--- Name: organizations_to_tsvector_idx2; Type: INDEX; Schema: public; Owner: -; Tablespace: 
---
-
-CREATE INDEX organizations_to_tsvector_idx2 ON organizations USING gin (to_tsvector('english'::regconfig, (authorized_official_last_name)::text));
-
-
---
--- Name: organizations_to_tsvector_idx3; Type: INDEX; Schema: public; Owner: -; Tablespace: 
---
-
-CREATE INDEX organizations_to_tsvector_idx3 ON organizations USING gin (to_tsvector('english'::regconfig, (authorized_official_first_name)::text));
-
-
---
--- Name: organizations_to_tsvector_idx4; Type: INDEX; Schema: public; Owner: -; Tablespace: 
---
-
-CREATE INDEX organizations_to_tsvector_idx4 ON organizations USING gin (to_tsvector('english'::regconfig, (authorized_official_middle_name)::text));
-
-
---
--- Name: organizations_to_tsvector_idx5; Type: INDEX; Schema: public; Owner: -; Tablespace: 
---
-
-CREATE INDEX organizations_to_tsvector_idx5 ON organizations USING gin (to_tsvector('english'::regconfig, (authorized_official_telephone_number)::text));
-
-
---
```

Comment:
Now I understand your db/structure.sql question from earlier; the schema in this file  should reflect our actual schema, but this commit removes some items that really should be there, so should probably be reconsidered. It's auto generated based on running migrations, so sometimes it gets out of whack; the rule of thumb is to check it in if the change reflects an actual schema change you're trying to make, and otherwise not.